packages/storybook/src/stories/ino-snackbar/ino-snackbar.stories.ts

```diff
@@ -128,3 +130,34 @@ export const StayVisibleOnHover = template.generateStoryForProp(
       'This snackbar stays visible on hover otherwise it will disappear in 5s',
   },
 );
+
+/**
+ * To add a custom icon inside the `ino-snackbar` element, follow these steps:
+ * 1. Inside the `ino-snackbar` element, insert an `img` or a similar element.
+ * 2. Assign a `slot` attribute to the `img` element and set it to `"icon"`.
+ * 3. Ensure that the custom icon has the desired size. In the example below, we set the width and height of the icon to 25px.
+ *
+ * Example usage of the `ino-snackbar` element with a custom icon:
+ */
+export const CustomIcon = (args: InoSnackbarExtended) => html`
+  <ino-button class="snackbar-trigger" data-template-id="${args.id}-custom"
+    >Show Custom Snackbar
+  </ino-button>
+  <template id="${args.id}-custom">
+    <ino-snackbar
+      id="${args.id}-custom"
+      action-text="${args.actionText}"
+      timeout="${args.timeout}"
+      type="${args.type}"
```

Comment:
Maybe we have to extend `type` to hold custom strings associated with the custom icons. In Addition we can offer a css variable for setting the color of the border (and icon bg). With that the consumer can build their specific snackbar message which doesn't overlap with the given ones. (We have to discuss this with the team first)